src/webapp/components/dataset-wizard/FilterIndicators.tsx

```diff
@@ -6,20 +6,25 @@ import FilterListIcon from "@material-ui/icons/FilterList";
 import CloseIcon from "@material-ui/icons/Close";
 import i18n from "$/utils/i18n";
 import { Dropdown } from "@eyeseetea/d2-ui-components";
+import { CoreCompetency } from "$/domain/entities/DataSet";
+import _ from "$/domain/entities/generic/Collection";
 
-export type FilterType = "scope" | "core" | "outputType";
+export type FilterType = "scope" | "core" | "outputType" | "theme" | "group";
 
 export type FilterIndicatorsProps = {
-    coreCompetencies: string[];
-    coreValue: string;
+    coreCompetencies: CoreCompetency[];
+    coreValues: string[];
     groups: string[];
+    group: string;
     onClose: () => void;
-    onFilterChange: (scope: string, type: FilterType) => void;
+    onFilterChange: (scope: ChipItem | ChipItem[], type: FilterType) => void;
```

Comment:
typically, we try to avoid this JS (T | T[]) acceptance (we accept T[] and create another function that wrap the single case)